src/handler/global/start.ts

```diff
@@ -25,20 +25,24 @@ export class Start {
             .toJSON();
     }
 
-    /**
-     * @todo Add additional handling for invalid api key
-     */
     @Command({ name: "start" })
     @Guard({ body: commandChannelGuard })
-    async start(interaction: CommandInteraction): Promise<void> {
+    async start(interaction: CommandInteraction) {
         await interaction.deferReply();
         const apiKey = interaction.options.getString("apikey", true);
         if (apiKey.length != 10) {
             throw new BotError(
                 "API keys must be 10 characters, please check your key and try again."
             );
         }
-        const user = await getUser(apiKey);
+        let user;
+        try {
+            user = await getUser(apiKey);
+        } catch (e) {
```

Comment:
maybe it would be better to check for error 1 and differentiate between invalid keys and other errors that way